content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -278,7 +280,7 @@ You need a GitHub repository to add your project to Deno Deploy and enable autom
 ### Steps
 
 1. [Create a private GitHub repository](https://github.com/new).
-   ![Create a private repository](../../../doc-images/guides/deploy/create-repo.png)
```

Comment:
Why remove this?

Reply:
Removed that as the UI on Github's private repo may change. So, I ensured it had only the instruction without the image.